postcss.config.js

```diff
@@ -1,23 +1,23 @@
 const path = require('path')
 
-const { production } = require('./config/webpack.vars')
```

Comment:
It was broken since we moved from `NODE_ENV` to webpack env: `production` was always undefined as we didn't give any env to `webpack.vars`.